c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -752,6 +752,7 @@ $(TX_RC):
 .build/locale/%/LC_MESSAGES/gmf.po: $(TX_DEPENDENCIES)
 	$(PRERULE_CMD)
 	mkdir -p $(dir $@)
+	sleep 1 # Should wait on the creation of the directory
```

Comment:
=> `To be sure that the the creation date should be after the .transifexrc file. (the $(TOUCHBACK_TXRC) lost the ms)`